assets/travis-worker/clean-up-containers.bash

```diff
@@ -135,6 +130,27 @@ EOF
     "${LIBRATO_API}/v1/metrics"
 }
 
+__container_must_be_cleaned() {
+  local cid="${1}"
+  local max_age="${2}"
+
+  if __container_has_active_exec "${cid}" &&
+    __container_is_newer_than "${cid}" "${max_age}"; then
+    return 1
+  fi
+}
+
+__container_has_active_exec() {
+  local cid="${1}"
+  local exec_ids
+
+  exec_ids="$(docker inspect --format='{{ .ExecIDs | json }}' "${cid}")"
```

Comment:
```
$ docker inspect --format='{{ .ExecIDs | json }}' $(docker ps -ql)
["351c339e17afa8afdebc493e33245923a98ce6c72b998ce427ffb632712b3583"]
```

Neat, I didn't know about this! I'm curious what (if anything) we can do with `ExecID`; I'm not finding much documentation on it at first glance. 

Is this a reliable way to distinguish containers that no longer have any business running??

Reply:
My intent here was to use the presence of an `.ExecIDs` array to infer whether or not travis-worker is actively running the job, yep!